tests/jax/unit-tests.libsonnet

```diff
@@ -45,8 +45,7 @@ local mixins = import 'templates/mixins.libsonnet';
         exit 1
       fi
 
-      # b/192016388
-      %(maybeInstallTF)s
+      %(envWorkarounds)s
```

Comment:
testEnvWorkarounds

Reply:
Forgot to save this file in my editor :woman_facepalming: Fixed now.